app/org/thp/cortex/util/docker/DockerClient.scala

```diff
@@ -0,0 +1,167 @@
+package org.thp.cortex.util.docker
+
+import com.github.dockerjava.api.model._
+import com.github.dockerjava.core.{DefaultDockerClientConfig, DockerClientConfig, DockerClientImpl}
+import com.github.dockerjava.transport.DockerHttpClient
+import com.github.dockerjava.zerodep.ZerodepDockerHttpClient
+import play.api.{Configuration, Logger}
+
+import java.nio.file.{Files, Path}
+import java.time.Duration
+import java.util.concurrent.Executors
+import scala.concurrent.duration.FiniteDuration
+import scala.jdk.CollectionConverters._
+import scala.util.Try
+
+class DockerClient(config: Configuration) {
+  private lazy val logger: Logger           = Logger(getClass.getName)
+  private lazy val (dockerConf, httpClient) = getHttpClient
+  private lazy val underlyingClient         = DockerClientImpl.getInstance(dockerConf, httpClient)
+
+  def execute(containerId: String): Try[String] = Try {
+    val startContainerCmd = underlyingClient.startContainerCmd(containerId)
+    startContainerCmd.exec()
+    val waitResult = underlyingClient
+      .waitContainerCmd(containerId)
+      .start()
+      .awaitStatusCode()
+    logger.info(s"container $containerId started and awaited with code: $waitResult")
+
+    containerId
```

Comment:
Returning the container ID is not very useful as it is in parameters. You can return the exit code.